### ISSUE TITLE
Confused about the vgg(cfg, i, batch_norm=False)  in ssd.py

### ISSUE THREAD

**Issue by sanersbug** [amdegroot/ssd.pytorch#302]
When  i  set the batch_norm = True, the train will report an error, if the parameter can work normal???
i just want to try the Batch Normalization after Convolution layer ,anyone know how to do it?

**Comment by ZhongyuanW** [comment]
I meet the same eorr. Did you solve it?
